src/server/apps/worldserver/worldserver.conf.dist

```diff
@@ -2602,6 +2602,15 @@ ItemDelete.Quality = 3
 
 ItemDelete.ItemLevel = 80
 
+#
+#
+#    ItemDelete.KeepDays
+#        Description: Time (in days)
+#        Default:     30 - (Enabled)
+#                     0  - (Disabled, Don't delete any items)
+
+ItemDelete.KeepDays = 30
```

Comment:
Default 0 to not leave anyone who didn't read this PR surprised when items are permanently deleted?